src/tools/stringSlice.go

```diff
@@ -20,3 +20,20 @@ func Deduplicate(ss []string) (output []string) {
 
 	return
 }
+
+func GetStringsNotInSlice(references []string, strings []string) []string {
```

Comment:
This is usually just called a `xor` operation. In this case it's just for string slices, but it could easily be extended to support any type with generics.

Which is probably not needed for this program, so I'd wager we could just use some kind of prefix since `[]string` can't implement interfaces.

How about
```suggestion
func StrSliceXor(references []string, strings []string) []string {
```

Reply:
Ok I will look at this

Reply:
Corrected